mbrl/env/ant_truncated_obs.py

```diff
@@ -39,17 +57,17 @@ def step(self, a):
     def _get_obs(self):
         return np.concatenate(
             [
-                self.sim.data.qpos.flat[2:],
-                self.sim.data.qvel.flat,
-                # np.clip(self.sim.data.cfrc_ext, -1, 1).flat,
+                self.data.qpos.flat[2:],
+                self.data.qvel.flat,
+                # np.clip(self.data.cfrc_ext, -1, 1).flat,
             ]
         )
 
     def reset_model(self):
         qpos = self.init_qpos + self.np_random.uniform(
             size=self.model.nq, low=-0.1, high=0.1
         )
-        qvel = self.init_qvel + self.np_random.randn(self.model.nv) * 0.1
+        qvel = self.init_qvel + self.np_random.standard_normal(self.model.nv) * 0.1
```

Comment:
is `self.model.nv` the right shape? My understanding is that standard_normal and randn take different shape formats?

Reply:
Can you clarify? The numpy doc seems like they're identical: https://numpy.org/doc/stable/reference/random/generated/numpy.random.randn.html

Reply:
Okay, that's clearer than what I found. You're right.